### PR TITLE
Bug#1452816 - Permissions for Cloud Provider account - fixes

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For - https://bugzilla.redhat.com/show_bug.cgi?id=1426755 and https://bugzilla.redhat.com/show_bug.cgi?id=1452816

### CODE REVIEW

File: install_config/configuring_gce.adoc [hunk 1]
```diff
@@ -18,6 +18,23 @@ volumes as persistent storage] for application data. After GCE is configured
 properly, some additional configurations will need to be completed on the
 {product-title} hosts.
 
+[[configuring-azure-permissions]]
```
Comment (pschiffe) [inline install_config/configuring_gce.adoc]: typo, this should be `configuring-gce-permissions`. Also, I'm afraid that the correct term here is GCP - Google Cloud Platform. GCE - Google Compute Engine is only a part of the GCP, and the OCP on GCP is using more parts of the GCP, for example Google Cloud Storage, which belongs to the GCP, not to the GCE.

File: install_config/configuring_aws.adoc [hunk 2]
```diff
@@ -18,20 +18,52 @@ volumes as persistent storage] for application data. After AWS is configured
...
+| _ec2:DescribeInstance_ | _elasticloadbalancing:DeleteLoadBalancerListeners_
+| _ec2:AttachVolume_ | _elasticloadbalancing:RegisterInstancesWithLoadBalancer_
+| _ec2:DetachVolume_ | _elasticloadbalancing:DescribeLoadBalancers_
+| _ec2:DeleteVolume_ | _elasticloadbalancing:CreateLoadBalancer_
+| _ec2:DescribeSubnets_ | _elasticloadbalancing:DeleteLoadBalancer_
+| _ec2:CreateSecurityGroup_ | _elasticloadbalancing:ModifyLoadBalancerAttributes_
+| _ec2:DescribeSecurityGroups_ | _elasticloadbalancing:DescribeLoadBalancerAttributes_
+| _ec2:DescribeRouteTables_ |
+| _ec2:AuthorizeSecurityGroupIngress_ |
+|===
```
Comment (sferich888) [inline install_config/configuring_aws.adoc]: These are very generic permissions, do we have anything more security restrictive?

File: install_config/configuring_azure.adoc [hunk 3]
```diff
@@ -19,6 +19,12 @@ Azure disk as persistent storage] for application data. After Azure is
 configured properly, some additional configurations need to be completed on the
 {product-title} hosts.
 
+[[configuring-azure-permissions]]
+== Permissions
+Configuring Azure for {product-title} requires following roles:
+
+* *Contributor* - To create and manage all types of Azure resources.
```
Comment (sferich888) [inline install_config/configuring_azure.adoc]: This are very generic permissions, do we have anything more security restrictive?

File: install_config/configuring_gce.adoc [hunk 4]
```diff
@@ -18,6 +18,23 @@ volumes as persistent storage] for application data. After GCE is configured
...
+== Permissions
+Configuring GCE for {product-title} requires following role:
+
+* *roles/owner* - To create service accounts, cloud storage, instances, images, templates, Cloud
```
Comment (sferich888) [inline install_config/configuring_gce.adoc]: These are very generic permissions, do we have anything more security restrictive?

File: install_config/configuring_openstack.adoc [hunk 5]
```diff
@@ -15,6 +15,13 @@ When deployed on link:https://www.openstack.org/[OpenStack], {product-title} can
 be configured to access OpenStack infrastructure, including
 xref:../install_config/persistent_storage/persistent_storage_cinder.adoc#install-config-persistent-storage-persistent-storage-cinder[using OpenStack Cinder volumes as persistent storage] for application data.
 
+[[configuring-opestack-permissions]]
+== Permissions
+Configuring OpenStack for {product-title} requires following permissions:
+
+* *`_member_`* - For creating assets, you need the member role on the project.
```
Comment (sferich888) [inline install_config/configuring_openstack.adoc]: Do we outline anything about LBaaS? And what needs to be installed for router ingress?

**Comment by cooktheryan** [comment]
+1 on the AWS section

**Comment by gaurav-nelson** [comment]
Closing this one, will create a new PR with latest changes.

**Comment by vikram-redhat** [comment]
@gaurav-nelson - make sure to remove the milestones and labels once a PR is closed without merging.

**Pull request closed without merging**